aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/curating_good_algorithm_listing.md

```diff
@@ -0,0 +1,46 @@
+| H4  | Highlights       | Specify if algorithm is compatible with model auto tuning functionality, distributed training, GPU on Amazon SageMaker. | Mandatory                    | The model can be trained using automatic model tuning capability and you can specify multiple instances while running a model training job                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| PO3 | Product overview | List the core framework that the model/algorithm was built on                                                           | Highly Recommended           | Unlike existing AutoML frameworks that primarily focus on model/hyperparameter selection, AutoGluon-Tabular succeeds by ensembling multiple models and stacking them in multiple layers.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| PO4 | Product overview | Differentiated capabilities of model/algorithm                                                                          | Mandatory                    | Dynamic factor models (DFMs) can be used to analyze and forecast large sets of time-series, such as measurements and indicators of national or multinational economies, prices of products or instruments constantly traded in markets, measurements and observations of natural or engineering processes, and trends in social media or sports tournaments, etc. <br>The evolutions of these time-series are influenced by evolutions of a number of unobserved factors commonly affecting all or many of the time-series. A long-memory DFM can estimate influences of longer histories of factor evolutions.                                                                                                                                                                                                                               |
```

Comment:
the "such as" covers the "etc.". remove:
```suggestion
| PO4 | Product overview | Differentiated capabilities of model/algorithm                                                                          | Mandatory                    | Dynamic factor models (DFMs) can be used to analyze and forecast large sets of time-series, such as measurements and indicators of national or multinational economies, prices of products or instruments constantly traded in markets, measurements and observations of natural or engineering processes, and trends in social media or sports tournaments. <br>The evolutions of these time-series are influenced by evolutions of a number of unobserved factors commonly affecting all or many of the time-series. A long-memory DFM can estimate influences of longer histories of factor evolutions.                                                                                                                                                                                                                               |
```

Reply:
The values in _sample example_ are copied from AWS Marketplace listings as it is